cmd/helm/show.go

```diff
@@ -60,18 +60,17 @@ func newShowCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 	client := action.NewShowWithConfig(action.ShowAll, cfg)
 
 	showCommand := &cobra.Command{
-		Use:               "show",
-		Short:             "show information of a chart",
-		Aliases:           []string{"inspect"},
-		Long:              showDesc,
-		Args:              require.NoArgs,
-		ValidArgsFunction: noCompletions, // Disable file completion
```

Comment:
Because this is a command with sub-commands, we don't need to define a `ValidArgsFunction` for it as Cobra will automatically complete the sub-commands and turn off file completion.

This wasn't the case when shell completion was first implemented for this command which is why we had to use `ValidArgsFunction`.